components/SideMenu.tsx

```diff
@@ -19,6 +19,16 @@ function SideMenu(props: { handleDrawerToggle: () => void }) {
     },
   ];
 
+  const linksComponent = sideLinks.map((link, index) => (
+    <li>
+      <Link href={link.path} key={index} rel="noreferrer" className={`block py-2 px-3 w-fit m-2 text-left font-bold rounded-md`} aria-label="Github">
```

Comment:
remove `key={index}`

```suggestion
      <Link href={link.path}  rel="noreferrer" className={`block py-2 px-3 w-fit m-2 text-left font-bold rounded-md`} aria-label="Github">
```